# Calculating binomial probabilities
Just as in the last exercise, assume that Amir wins 30% of deals. He wants to get an idea of how likely he is to close a certain number of deals each week. In this exercise, you'll calculate what the chances are of him closing different numbers of deals using the binomial distribution.

In [3]:
import numpy as np
from scipy.stats import binom

success_rate = 0.3
deals_per_week = 3

- What's the probability that Amir closes all 3 deals in a week? Save this as prob_3.

In [4]:
prob_3 = binom.pmf(3, deals_per_week, success_rate)
prob_3

0.027

- What's the probability that Amir closes 1 or fewer deals in a week? Save this as prob_less_than_or_equal_1.

In [11]:
prob_less_than_or_equal_1 = binom.cdf(1, deals_per_week, success_rate)
prob_less_than_or_equal_1

0.784

- What's the probability that Amir closes more than 1 deal? Save this as prob_greater_than_1.

In [8]:
prob_greater_than_1 = 1 - binom.cdf(1, deals_per_week, success_rate)
prob_greater_than_1

0.21599999999999997

Powerful probability calculations! Amir has about a 22% chance of closing more than one deal in a week.